lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -526,6 +537,21 @@ generator (Model _ wids) = Just $ frequency $ fmap (fmap At) <$> concat
     genPrivKey :: Gen MPrivKey
     genPrivKey = elements ["pk1", "pk2", "pk3"]
 
+    genSortOrder :: Gen SortOrder
+    genSortOrder = QC.elements [Ascending, Descending]
```

Comment:
This can be replaced with `arbitraryBoundedEnum` from `Test.QuickCheck.Arbitrary`:

```hs
arbitraryBoundedEnum :: (Bounded a, Enum a) => Gen a 
```